### PR TITLE
Fix bundle generation

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Also fixes Holodeck file

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -119,8 +119,8 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build image
         env:
-          BUNDLE_IMAGE_NAME: ghcr.io/${LOWERCASE_REPO_OWNER}/k8s-nim-operator
-          BUNDLE_IMAGE: ghcr.io/${LOWERCASE_REPO_OWNER}/k8s-nim-operator:bundle-lastest-${{ github.ref }}
+          BUNDLE_IMAGE_NAME: ghcr.io/nvidia/k8s-nim-operator
+          BUNDLE_IMAGE: ghcr.io/nvidia/k8s-nim-operator:bundle-lastest${BRANCH}
```
Comment (shivamerla) [inline .github/workflows/image.yml:123]:
```suggestion
          BUNDLE_IMAGE: ghcr.io/nvidia/k8s-nim-operator:bundle-latest${BRANCH}
```

Comment (ArangoGutierrez) [reply .github/workflows/image.yml]: done

File: .github/workflows/image.yml [hunk 2]
```diff
@@ -102,10 +102,13 @@ jobs:
...
           echo "BUILD_MULTI_ARCH_IMAGES=${GENERATE_ARTIFACTS}" >> $GITHUB_ENV
       - name: Extract branch name for push events
-        if: github.event_name == 'push'
         id: extract_branch
         run: |
-          BRANCH=${GITHUB_REF#refs/heads/}
+          BRANCH_NAME=${GITHUB_REF#refs/heads/}
+          BRANCH="bundle-latest"
+          if [[ "$BRANCH_NAME" == "main" || "$BRANCH_NAME" == "release-"* ]]; then
+            BRANCH="bundle-lates-${BRANCH_NAME}"
```
Comment (shivamerla) [inline .github/workflows/image.yml:110]:
```suggestion
            BRANCH="bundle-latest-${BRANCH_NAME}"
```

Comment (ArangoGutierrez) [reply .github/workflows/image.yml]: fixed

**Review by shivamerla — APPROVED** [review]

**Pull request merged**